src/common-components/ThirdPartyAuth.jsx

```diff
@@ -20,20 +24,32 @@ import {
 const ThirdPartyAuth = (props) => {
   const { formatMessage } = useIntl();
   const {
-    providers, secondaryProviders, currentProvider, handleInstitutionLogin, thirdPartyAuthApiStatus,
+    providers, 
+    secondaryProviders, 
+    currentProvider, 
+    handleInstitutionLogin, 
+    thirdPartyAuthApiStatus, 
+    helpingText, 
```

Comment:
So as we know that this is either used by Login or Register page. Can we please use the `isLogin` prop to have a condition on what text to render? there is no need for this prop in my opinion.